apps/moderatorfeedback/serializers.py

```diff
@@ -26,3 +28,36 @@ def get_last_edit(self, statement):
         if statement.modified:
             return statement.modified
         return statement.created
+
+
+class CommentWithStatementSerializer(a4_serializers.CommentSerializer):
+
+    moderator_statement = ModeratorCommentStatementSerializer(read_only=True)
+
+    class Meta:
+        model = Comment
+        read_only_fields = \
+            a4_serializers.CommentSerializer.Meta.read_only_fields + \
+            ('moderator_comment_statement',)
+        exclude = ('creator',)
+
+
+class CommentWithStatementListSerializer(CommentWithStatementSerializer):
+    """Serializer for the comments to be used when viewed as list."""
+
```

Comment:
Does it work that empty? No `pass` or anything like that?

Reply:
I was also wondering, just copied that as it is from A4 and it seems to work, yes :woman_shrugging: 
https://github.com/liqd/adhocracy4/blob/84d56f6479d7072f15a4b06011d87f41027e8ca1/adhocracy4/comments_async/serializers.py#L202

Reply:
Well, then. Ehm. Let's keep it like that? Or try if it still works when we do add the `pass`? Somehow looking at it makes me feel slightly uneasy. :rofl: